python-qa-tools/utils/commands.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+
+def command_processor(command):
```

Comment:
```suggestion
def exec_cmd(cmd):
```

Reply:
add some description for each method please

Reply:
Replaced as mentioned.

Reply:
I can't see the update here